lib/src/message/message_builder.dart

```diff
@@ -247,6 +313,32 @@ class _MessagePartImageAttachmentState extends State<MessagePartImageAttachment>
     );
   }
 
+  void loadThumbnail() async {
+    imageProvider = MemoryImage(kTransparentImage);
+    String videoPath = _getMessageStateData(context).attachmentStateData.path;
+    String thumbnailPath = "";
+    int dotIndex = videoPath.lastIndexOf('.');
```

Comment:
Looks like something we should put in a helper or research if there is already a getFileExtension method in Dart iself.

Reply:
done in https://github.com/open-xchange/ox-coi/pull/298/commits/da49c93d52a3c3b67204f870a88b4cfe2e5123eb